README.md

```diff
@@ -1,3 +1,42 @@
 # Compatibility layer
 
+## Introduction
+
 The compatibility layer of the EESSI project uses [Gentoo Prefix](https://wiki.gentoo.org/wiki/Project:Prefix)
+to provide a known base on top of the host. This is the foundation we use to build our software stack on.
+An alternative would be the NixOS.
+
+## Installation and Configuration
+
+### Prerequisites
+
+The bootstrap process will need a clean environment with a compiler (the system version of gcc will do). It also is very sensitive to 
+the environment, so setup a user with unset CFFLAGS, CFLAGS, LDFLAGS, PKG_CONFIG_PATH and the always harmful LD_LIBRARY_PATH variables.
+EESSI provides a singularity container for this.
+
+## Installation and Configuration
```

Comment:
There are two section named "Installation and Configuration". Probably best to merge them by moving the subsection Prerequisites to the second section?